### DISCUSSION TITLE
Sicurezza content

### DISCUSSION THREAD

**Discussion by dleone81** [dleone81/apcelettro#41] [General]
Mi serve avere il contenuto della pagina

**Comment by Cattalek93** [comment]
lascerei un attimo in standby e aggiornemo quando ci uniremo

**Comment by dleone81** [comment]
Ciao @Cattalek93 , @fabiocrazycurls  ho rimosso il contenuto dal menu, quando sarà pronto il contenuto lo re-inserirò
